README.md

```diff
@@ -1 +1,85 @@
-# CapstoneJS
\ No newline at end of file
+# CapstoneJS
+
+In this project we will prepare a Kanban board with a GitHub project, with my Learning partner. It's javascript capstone project with external API..
+
+![image] ![screenshot](photo.jpg)
+
+## Video link
+
+[loom Link] (https://loom.com/share/cebf106ada4f4ac789399a6eb27dd385)
+
+## Live Demo
+
```

Comment:
Please add an active live demo link or remove this option from here.

Reply:
Thank you.

Reply:
Thank you